custom_components/mawaqit/manifest.json

```diff
@@ -1,9 +1,9 @@
 {
   "domain": "mawaqit_prayer_times",
   "name": "Mawaqit Prayer Times",
-  "documentation": "https://github.com/mawaqit/home-assistant",
-  "requirements": ["mawaqit_times_calculator>=0.0.4","pymysql"],
-  "codeowners": ["@mawaqit"],
+  "documentation": "https://www.home-assistant.io/integrations/mawaqit_prayer_times",
+  "requirements": ["mawaqit_times_calculator>=1.0.2"],
+  "codeowners": ["@iyac69"],
```

Comment:
can you put @mawaqit organization as codeowners ?

Reply:
sure. I will change it in the new push

I did slight changes in the forthcoming version. the update of the sensor is done a minute after each prayer call. so that, we do not wait midnight to start showing the next prayers. after Asr pray, the sensor Asr is updated by the Asr of the next day

it is doable now since we store the whole year calendar !